views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
 
-          p Ethereum is a &nbsp;
-            strong decentralized platform that runs smart contracts
-            | : applications that run exactly as programmed without any possibility of downtime, censorship, fraud or third party interference.
+          p Ethereum is an open-source, public, blockchain-based distributed computing platform featuring smart contracts that
+            | facilitate online agreements. The Ethereum Virtual Machine (EVM)
+            | executes scripts on a decentralized network of servers (nodes) and is fueled by its virtual currency (ether).
```

Comment:
The EVM bit is a bit hard core for non-devs.   I welcome ideas here, and perhaps after this is up and available others will have ideas for improving on it.